nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -132,49 +135,85 @@ def _get_input_edges_mapping(nncf_graph: NNCFGraph):
         return get_input_edges_mapping(nncf_graph)
 
     @staticmethod
-    def _get_axis(
-        nncf_graph: NNCFGraph, target_point: ONNXTargetPoint, quantizer_config: QuantizerConfig
+    def _get_quantized_channel_axis(
+        is_per_channel: bool,
+        target_point: ONNXTargetPoint,
+        node: NNCFNode,
     ) -> Optional[int]:
-        if not quantizer_config.per_channel:
-            return None
-        if not target_point.is_weight_target_point():
+        if is_per_channel:
+            if target_point.is_weight_target_point():
+                return ONNXMinMaxAlgoBackend._get_weight_channel_axis(node, target_point.port_id)
+            # Channel axis for activation
             return 1
-        node = nncf_graph.get_node_by_name(target_point.target_node_name)
+        return None
 
+    @staticmethod
+    def _get_weight_channel_axis(node: NNCFNode, port_id: int) -> int:
+        """
+        Returns channel axis for a weight on port_id for a node.
+
+        :param node: NNCFNode, which has a weight on input port_id.
+        :param port_id: Input port id on which there is a weight of a node.
+        :return: Weight channel axis.
+        """
         weight_channel_axis = node.metatype.weight_channel_axis
         if node.layer_attributes.has_node_attrs():
             if node.metatype == om.ONNXGemmMetatype:
-                weight_shape = node.layer_attributes.weight_attrs[target_point.port_id]["shape"]
+                weight_shape = node.layer_attributes.weight_attrs[port_id]["shape"]
                 if (
-                    target_point.port_id == 0
+                    port_id == 0
                     and node.layer_attributes.node_attrs["transA"] == 1
-                    or target_point.port_id == 1
+                    or port_id == 1
                     and node.layer_attributes.node_attrs["transB"] == 1
                 ):
                     weight_channel_axis = transpose_axis(weight_shape, weight_channel_axis)
         return weight_channel_axis
 
     @staticmethod
-    def _get_reduction_shape_and_use_abs_max(
-        nncf_graph: NNCFGraph, target_point: ONNXTargetPoint, quantizer_config: QuantizerConfig
-    ) -> Tuple[Optional[Tuple[int, ...]], bool]:
-        use_abs_max = quantizer_config.mode == QuantizationMode.SYMMETRIC
-        if not quantizer_config.per_channel:
-            return None, use_abs_max
+    def _get_reduction_shape_for_weight(
```

Comment:
Wouldn't we need this method in `node_utils.py`? For the `ChannelAlignment` or `SmoothQuant` algorithms, for example.
Also, no description of method.